tests/integration/interactive_add_binding_test.go

```diff
@@ -224,12 +224,12 @@ var _ = Describe("odo add binding interactive command tests", func() {
 			var nsWithNoService string
 
 			BeforeEach(func() {
+				nsWithNoService = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
 				otherNS = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
 				addBindableKindInOtherNs := commonVar.CliRunner.Run("-n", otherNS, "apply", "-f",
 					helper.GetExamplePath("manifests", "bindablekind-instance.yaml"))
 				Expect(addBindableKindInOtherNs.ExitCode()).To(BeEquivalentTo(0))
-
-				nsWithNoService = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
+				commonVar.CliRunner.HasNamespaceProject(nsWithNoService)
```

Comment:
`HasNamespaceProject` returns a Boolean. It might make sense to check the return value.

Reply:
updated